spec/index.bs

```diff
@@ -1901,6 +1908,7 @@ interface GPUSupportedLimits {
     readonly attribute unsigned long maxComputeWorkgroupSizeY;
     readonly attribute unsigned long maxComputeWorkgroupSizeZ;
     readonly attribute unsigned long maxComputeWorkgroupsPerDimension;
+    readonly attribute unsigned long maxClipDistances;
```

Comment:
Nit: Should it be renamed to `maxClipDistancesSize` then to make it clear it's about the size of the array, not the value of clip distances?